### ISSUE TITLE
What are the GO terms doing here?

### ISSUE THREAD

**Issue by caufieldjh** [berkeleybop/artificial-intelligence-ontology#63]
I first saw `biological process` at the root - see https://bioportal.bioontology.org/ontologies/AIO/?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0008150#details

Then saw that three different GO terms are present in AIO: 
GO:0003674
GO:0008150
GO:0016301

These don't connect to anything.

**Comment by caufieldjh** [comment]
They're coming in from SWO:
```
~/artificial-intelligence-ontology$ grep "# Class: <http://purl.obolibrary.org/obo/GO_" ./src/ontology/imports/swo_import.owl
# Class: <http://purl.obolibrary.org/obo/GO_0003674> (molecular_function)
# Class: <http://purl.obolibrary.org/obo/GO_0008150> (biological_process)
# Class: <http://purl.obolibrary.org/obo/GO_0016301> (kinase activity)
```

**Comment by caufieldjh** [comment]
The terms don't appear to be doing much in SWO, either (e.g. https://bioportal.bioontology.org/ontologies/SWO/?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0008150)
We can safely exclude them.

**Comment by realmarcin** [comment]
Just catching up here -- what is the best way to exclude? Yes these are showing up in Bioportal viz...

**Comment by caufieldjh** [comment]
In phenio there's a file of explicitly excluded terms - see https://github.com/monarch-initiative/phenio/blob/main/src%2Fontology%2Fexclude-terms.txt

**Comment by caufieldjh** [comment]
Then a ROBOT filter command like here: https://github.com/monarch-initiative/phenio/blob/main/src%2Fontology%2Fphenio.Makefile#L72

**Comment by caufieldjh** [comment]
Resolved by #82
